index.js

```diff
@@ -29,6 +29,10 @@ class Library {
 }
```

Comment:
[ OPTIONAL BUT RECOMMENDED ] After adding the books to the list when the user refreshes the page list becomes empty. Please make sure that your data is preserved in the browser and do not get lost on refresh.

Reply:
We have tried to implement this with some level of success